README.md

```diff
@@ -34,6 +34,8 @@ For the **desktop build** (currently, the only supported platform) do not pass t
 
 ### Example: Build a Docker image with Server v4.20.1
 
+**NB:** On a new Ubuntu release you must update the [`setup_jellyfin_repo.sh`](./setup_jellyfin_repo.sh) shell script for `jellyfin-ffmpeg`.
```

Comment:
eh? but you deleted this script with this PR

Reply:
Yes, you are correct. We will remove this part in the README.